packages/notifi-dapp-example/src/app/notifi/dashboard/page.tsx

```diff
@@ -23,7 +23,9 @@ export default function NotifiDashboard() {
 
   let accountAddress: string | undefined = '';
   if (selectedWallet) {
-    accountAddress = wallets[selectedWallet].walletKeys?.bech32;
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    accountAddress = wallets[selectedWallet].walletKeys.hex ?? '';
```

Comment:
We might want to handle the different case that 
- the accountAddress is Bech32 when `Cosmos` chain
- the accountAddress is Hex when `EVM` chain

Reply:
fixed.